docs/rpc_aio_leapfrog.md

```diff
@@ -50,12 +52,14 @@ need to be layered on top of RPC-O to test with. (i.e. swift, ceph, etc)
 
 ##### Ztrigger
 
-Determines the types of tests to be ran, periodically ran, or tested on PR from
-the specified branches in the series.
+This determines what triggers the job, a PR via GitHub (github-pull-request) or a
+periodic triggered by the cron scheduled passed to *timed*. Periodic jobs can be
+started on demand and allow the parameters to be tuned before running and is used
+to validate that the code in the branch is still valid and passes.
```

Comment:
Do you want to add emphasis around `github-pull-request`?  Also, I'd drop the bit about on-demand as both can technically be used to spin up on-demand jobs (tho the periodics are slightly easier).